pkg/doc/verifiable/credential.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 
 	"github.com/hyperledger/aries-framework-go/pkg/common/log"
-	"github.com/hyperledger/aries-framework-go/pkg/doc/jwt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose/jwt"
```

Comment:
I'm not sure why this folder was moved as part of the JWE PR?
@kdimak @DRK3 @Baha-sk

Reply:
The initial idea was to have `jose` package and put inside it the implementation of JWS and JWE. JWT is a bit different thing (it uses JWS and JWE to build claims) and it was placed in a separate package. Likewise, we are gonna put JWM implementation in a separate (non `jose`) package.

Reply:
I was thinking that JWT should go here since it's part of the JOSE framework of standards - my thinking is that if we have a JOSE folder, then all the JOSE stuff should go under there. Otherwise, if JWT is separate, then I think JWE and JWS should be separate too for consistency. What do you think? @kdimak @Baha-sk @troyronda

Reply:
@DRK3 Jose is both signing and encryption. Keep those together as `jose`.
jwt uses jose to produce JWS and JWE versions. 
Regardless moving unrelated folders shouldn't be in this PR.

Note, we will soon have `jwm` as a sibling to `jwt`. I'm not sure they are logically child folders of jose.

Reply:
I agree with @troyronda . Maybe if we want to be consistent regarding a separation, we could also have `jose`, `jws` and `jwe` packages as @DRK3 suggested above. What do you think @troyronda ?

Reply:
@kdimak What would go under the `jose` package?

Reply:
keep jws and jwe under jose package, jwt and jwm can have their own package

Reply:
Done